Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.h

```diff
@@ -25,6 +25,7 @@
 @property (nonatomic, readonly) CGFloat initialHeight;
 @property (nonatomic, readonly) CGFloat buttonsViewHeight;
 @property (nonatomic) BOOL isFullScreenAvailable;
+@property (nonatomic) BOOL isContentHeightChangable;
```

Comment:
Really not sure the problem with the scroll is in the base class. You need to fix the specific screen instead of changing the entire base class